docs/learn/learn-auction.md

```diff
@@ -34,11 +34,21 @@ When candle auctions are used online, they require a random number to decide the
 termination.
 
 Parachain slot auctions differ slightly from a normal candle auction in that it does not use
-the random number to decide the duration of its opening phase. Instead, it has a *known open phase*
-and will be retroactively determined (at the normal close) to have ended at some point in the past
-during the ending phase. So during the open phase, bids will continue to be accepted, but later bids
-have higher probability of losing since the retroactively determined close moment may be found to
-have preceded the time that a bid was submitted.
+the random number to decide the duration of its opening phase. 
+
+The candle auction on Polkadot is split into two main parts: the *opening period* which is in 
```

Comment:
To keep things uniform across the websites - https://polkadot.network/auctions/
```suggestion
The candle auction on Polkadot is split into two parts: the *starting period* which starts 
```

Reply:
Opening period would be more accurate in auction terms. I believe it is referenced in the code as such too. Seems we should become uniform 
![image](https://user-images.githubusercontent.com/25497083/155256403-c446dd0e-d500-47b8-a933-9e2f1a381f69.png)

Reply:
The opening period is in play right after the auction "starts", but that start (as in, the official start of the auction) is an instant of time, so "starting period" doesn't seem as accurate

Reply:
Sounds good. Then this https://polkadot.network/auctions/ link will need an update